### PR TITLE
Handle redirect for /task-inspector and /one-click-loaner

### PR DESCRIPTION
**Pull request by helfi92**

### CODE REVIEW

File: src/App/index.js [hunk 1]
```diff
@@ -152,6 +152,16 @@ export default class App extends React.Component {
   render() {
     const { credentials, credentialsExpiringSoon } = this.state;
 
+    const PreviousAPI = (props) => {
```
Comment (eliperelman) [inline src/App/index.js]: Let's move this to outside of render.

File: src/App/index.js [hunk 2]
```diff
@@ -168,12 +178,13 @@ export default class App extends React.Component {
 
           <Grid fluid id="container">
             <Switch>
+              <PropsRoute path="/task-inspector" component={PreviousAPI} />
```
Comment (eliperelman) [inline src/App/index.js]: Add one for `/one-click-loaner`.

Comment (helfi92) [reply src/App/index.js]: Done

File: src/App/index.js (hunk 1 above)
Comment (helfi92) [reply src/App/index.js]: Done

**Review by eliperelman — APPROVED** [review]

**Pull request merged**